src/mondeath.cpp

```diff
@@ -844,3 +844,14 @@ void mdeath::fireball( monster &z )
         normal( z );
     }
 }
+
+
+void mdeath::conflagration( monster &z )
+{
+    for( auto &&dest : g->m.points_in_radius( z.pos(), 1 ) ) {
```

Comment:
Why an rvalue reference? I'd replace with a const reference: `for( const auto &dest ..`.